libraries/platform/src/platform/backend/LinuxPlatform.cpp

```diff
@@ -11,11 +11,11 @@
 
 #include <thread>
 #include <string>
-#include <CPUIdent.h>
+#include "../../CPUIdent.h"
 
 #include <QtCore/QtGlobal>
 
-#include <GPUIdent.h>
+#include "../../GPUIdent.h"
 #include <QSysInfo>
```

Comment:
Include statements should be sorted and grouped - coding standard 2.2.2.
This isn't always followed but it would be preferable to maintain at least some level ordering.
Here and elsewhere.

Reply:
Reordered them grouping in this order: standard library - system libraries - Qt libraries - other external libraries - Vircada libraries - local headers, keeping the header file that corresponds to the source at the top, as it seems to be the case in other places, and is a good practice IMO. 
It got a bit weird when pre-processor directives were involved, so not sure if I handled that correctly 
https://github.com/vircadia/vircadia/blob/7629fc3e48bafe2b8bdcf0f41818d48b2b2e7161/libraries/platform/src/platform/backend/MACOSPlatform.cpp#L9-L30
just grouped disregarding the directives (repeating them if necessary)

Reply:
Well that just broke everything .-.